api/src/multiaccessy/accessy.py

```diff
@@ -470,30 +509,20 @@ def _get_user_details(self, user_id: UUID) -> Any:
         return self._get(f"/org/admin/user/{user_id}", err_msg="Getting user details")
 
     def _get_users_org(self) -> list[dict]:
-        """Get all user ID:s"""
-
-        def is_application(user: dict) -> bool:
-            return user.get("msisdn", None) is None
+        """Get all users"""
 
         return [
             v
-            for v in self._get_json_paginated(f"/asset/admin/organization/{self.organization_id()}/user")
-            if not is_application(v)
+            for v in self._get_json_paginated(f"/org/organization/{self.organization_id()}/user")
+            if not v["application"]
```

Comment:
Fyi. This will now also include all accounts which have been reset.

I think accessy handles reset accounts by leaving them in place, but clearing their msisdn value.

Reply:
I just double-checked this by:
1. Registering a new user in Accessy
2. The user joined the Makerspace organization
3. The user then reset their account

After reset, they are not included in the API response.